### PR TITLE
fix: disable native-tls on rusoto

### PR DESCRIPTION
**Pull request by rkrasiuk**
Disable default native-tls feature on rusoto

### CODE REVIEW

**Review by mattsse — APPROVED** [review]

File: ethers-signers/Cargo.toml [hunk 1]
```diff
@@ -60,5 +60,5 @@ futures = ["futures-util", "futures-executor"]
 celo = ["ethers-core/celo"]
 ledger = ["coins-ledger", "futures", "semver"]
 yubi = ["yubihsm"]
-aws = ["rusoto_core", "rusoto_kms", "tracing", "spki"]
+aws = ["rusoto_core/rustls", "rusoto_kms/rustls", "tracing", "spki"]
```
Comment (gakonst) [inline ethers-signers/Cargo.toml:63]: leave this unchanged

File: ethers-signers/Cargo.toml [hunk 2]
```diff
@@ -34,8 +34,8 @@ trezor-client = { version = "0.0.7", optional = true, default-features = false,
 ] }
 
 # aws
-rusoto_core = { version = "0.48.0", optional = true }
-rusoto_kms = { version = "0.48.0", optional = true }
+rusoto_core = { version = "0.48.0", default-features = false, optional = true }
```
Comment (gakonst) [inline ethers-signers/Cargo.toml:37]:
```suggestion
rusoto_core = { version = "0.48.0", default-features = false, optional = true, features = ["rustls"] }
```

Comment (gakonst) [reply ethers-signers/Cargo.toml:37]: should be like this i think?

**Review by gakonst — CHANGES_REQUESTED** [review]
i think the way you did it, it would actually not work

**Review by gakonst — APPROVED** [review]
looks like they are the same

**Pull request merged**